dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -698,7 +698,7 @@ HRESULT CFSDropTarget::_DoDrop(IDataObject *pDataObject,
         _ILFreeaPidl(apidl, lpcida->cidl);
         ReleaseStgMedium(&medium);
     }
-    else if (SUCCEEDED(pDataObject->QueryGetData(&fmt2)))
+    else if (SUCCEEDED(hr = pDataObject->QueryGetData(&fmt2)))
```

Comment:
Same comment for this one as well. I bet that if you were instead to initialize the `hr` with the result from `pDataObject->GetData(&fmt2, &medium)` below instead, clang would be happy.